Widgets_CDAT/DictWidget/test_keyValue.py

```diff
@@ -0,0 +1,76 @@
+from PySide.QtCore import *
+from PySide.QtGui import *
+from collections import OrderedDict
+from dict import DictEditor
+
+d_e = None
+
+def dictEmitted(dict):
```

Comment:
Using `dict` as an argument is also a bad practice, for the same reason as naming the module that; it creates ambiguity as to what you're referring to when you type `dict`, which in most instances will mean the type `dict`, but here will be an instance of that type.